### DISCUSSION TITLE
Any reason why I can't trace this cable? Or have I modelled Krone LSA-PLUS incorrectly?

### DISCUSSION THREAD

**Discussion by chrisbright** [netbox-community/netbox#8902] [Q&A] [answered]
version is: v2.11.10, running in docker.

I have some connections where I cannot trace the full path, the trace from either A or B side ends at two patch panels which are then connected front to front with a patch.  The patch cable shows A and B points correctly, but when traced shows nothing.

We have a Krone LSA-PLUS distribution frame, I have modelled each block as a patch panel, with the top side of the block as 'rear ports' and the bottom as 'front ports', since the application is pretty much the same. 

The top side (rear) is permanent, either to a network switch or and to a patch panel located in a rack, and the bottom side (front) is always patched to another block within the Krone frame.

I have 8 frames with 36 blocks in each frame, so I modelled the frames as "Racks" and each block occupies 1U of the rack.  I think it would be the same as having 8 racks of 36 6-port patch panels.  Maybe there is a different way of doing this though?

Each block has 6 ports front and back, so all ports are 1 to 1, no splits or splices.

The path is:
Intf (Server) -> Patch Panel (front) - Patch Panel (rear) -> Krone Block 1 (rear) - Krone Block 1 (front) -> Krone Block 2 (front) - Krone Block 2 (rear) -> Intf (Switch)

From the Device to the Krone

![image](https://user-images.githubusercontent.com/13528103/158750633-915d9ba8-4d9d-4148-8d69-d462b0faa0ab.png)

From the Switch to the Krone
![image](https://user-images.githubusercontent.com/13528103/158754344-1c3d3b64-d158-49f5-a458-20b62a2d46c7.png)

The patch cable shown correctly landing on the bottom (front port) of both blocks.
![image](https://user-images.githubusercontent.com/13528103/158754442-30a59fbb-c6d5-435c-8677-8898fa96b66c.png)

Thanks for any ideas or suggestions how to achieve this!

**Comment by martinum4** [comment]
> The path is:
Intf (Server) -> Patch Panel (front) - Patch Panel (rear) -> Krone Block 1 (rear) - Krone Block 1 (front) -> Krone Block 2 (front) - Krone Block 2 (rear) -> Intf (Switch)

Each `->`is mapped a cable, right?

To make it more clear let me use your interface names again:
(start of first picture)
Juniper Eth1
Cable
FP 1/1 Rack 104
RP 1/1 Rack 104
Cable
RP B13 
(End of first picture)
(start of last picture)
FP B13
Cable
FP B18
(end of last picture)
(start of middle picture)
RP B18-1
BCAST C19
(end of middle picture)

is this correct?


When you click on the individual front ports are all these ports connected to the rear ports like this:
![grafik](https://user-images.githubusercontent.com/18295403/159651903-49843bec-3347-49e2-99d4-2f79d732e8d2.png)?
Do the rear ports have multiple positions?

**Comment by chrisbright** [comment, marked as answer]
Hi, @martinum4 
Thanks for helping! Yes your prephrasing is correct, and no, the rear ports have only single positions, and the front ports show their matching rear ports.  I also tried to create a full path by hand (rather than csv import) and it also looked the same, same not connected message, and tracing from either end stopped at the front/back ports of the krone block.

However I have just resolved the issue this evening by deleting all the front ports/rear ports and recreating them, except I swapped the the blocks around. Now the tops are the front port and the bottom (where the interconnect jumpers are) are the 'back ports'. 

Now it is:
Intf (Server) -> Patch Panel (front) - Patch Panel (rear) -> Krone Block 1 (front) - Krone Block 1 (rear) -> Krone Block 2 (rear) - Krone Block 2 (front) -> Intf (Switch)

I don't really see why this should be any different, but it works now!

I wonder if it was how the ports were created, since I created everything from a CSV import, since there were 36 blocks and 8 frames. 6 'ports' per block. Some 3,400 ports  Just leaving this here in case anyone else encounters this behavior...

I might try to recreate the behavior on another system to double check that it's not something I did... thanks again!
